docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -135,11 +135,20 @@ The `octopusdeploy/worker-tools` images provided for the execution containers fe
 
 The following worker-tools images are cached:
 
-- [Latest Windows-based image](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019): `!docker-image <octopusdeploy/worker-tools:windows.ltsc2019>`
+**On Linux Workers**:
 
 - [Latest Linux-based image](https://github.com/OctopusDeploy/WorkerTools/blob/master/ubuntu.18.04): `!docker-image <octopusdeploy/worker-tools:ubuntu.18.04>`
+- octopusdeploy/worker-tools:3.3.1-ubuntu.18.04
+- octopusdeploy/worker-tools:3.2.1-ubuntu.18.04
+- octopusdeploy/worker-tools:3.2.0-ubuntu.18.04
+- octopusdeploy/worker-tools:3.0.0-ubuntu.18.04
+
+**On Windows Workers**:
+
+- [Latest Windows-based image](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019): `!docker-image <octopusdeploy/worker-tools:windows.ltsc2019>`
+- octopusdeploy/worker-tools:3.3.1-windows.ltsc2019
```

Comment:
Should we remove this and add it later when the windows image is cached as well?